opcua/server/address_space.py

```diff
@@ -108,12 +108,13 @@ def _is_suitable_ref(self, desc, ref):
     def _suitable_reftype(self, ref1, ref2, subtypes):
         """
         """
+        if not subtypes and ref2.Identifier == ua.ObjectIds.HasSubtype:
+            return False
         if ref1.Identifier == ref2.Identifier:
             return True
-        #TODO: Please check the changes here
```

Comment:
not that change is correct. Looking at the test, I think the test was originally wrong.